smap_l2_gridder/collections.py

```diff
@@ -0,0 +1,124 @@
+"""Collection information.
+
+Defines hierarchies necessary for gridding collections with the SMAP L2 Gridder.
+
+While the input files are hierarchical and in hdf format, they are not fully
+self describing. Each griddable variable is tied to two variables that contain
+the column and row of a grid into which the value should be placed. But these
+column and row indices are not always in the same location.  Additionally,
+there are a number of different resolution grids and coordinate reference
+systems, that here are identified by a gpd file and epsg code that are
+described in the documentation for the dataset but are hard/impossible to
+determine by looking at the files themselves, for this reason we explictly lay
+out this information in this file along with helper routines to ease access.
+
+"""
+
+from .exceptions import InvalidCollectionError
+
+STANDARD_LOCATIONS = {
+    'row': 'Soil_Moisture_Retrieval_Data/EASE_row_index',
+    'col': 'Soil_Moisture_Retrieval_Data/EASE_column_index',
+}
+
+GRIDS = {
```

Comment:
This just links the grids with their EPSG for simplicity, in a perfect world they would be in the gpd file.